code/weapon/weapons.cpp

```diff
@@ -209,6 +209,9 @@ int		Weapon_impact_timer;			// timer, initialized at start of each mission
 //default time of a homing weapon to not home
 #define HOMING_DEFAULT_FREE_FLIGHT_TIME	0.5f
 
+// default percentage of max speed to coast at during freeflight
+#define HOMING_DEFAULT_FREE_FLIGHT_FACTOR	0.25f
```

Comment:
Please write as a constant value:
```suggestion
const float HOMING_DEFAULT_FREE_FLIGHT_FACTOR = 0.25f;
```